app/views/orders/cart.html.erb

```diff
@@ -1,43 +1,46 @@
-<section class="cart-page-container">
-  <section class="cart-header">
-    <h1>Shopping cart</h1>
-  <section>
+<h1>Shopping cart</h1>
+
+<% if @cart.orderitems.any? %>
+<table>
+  <tr>
+  <th>PHOTO</th>
+  <th>NAME</th>
+  <th>MERCHANT</th>
+  <th>QUANTITY</th>
+  <th>UNIT PRICE</th>
+  <th>LINE PRICE</th>
+  <th>REMOVE</th>
+  </tr>
+  <% sorted_items = @cart.orderitems.order(:id) %>
+  <% sorted_items.each do |item| %>
+  <tr>
+    <td><%= link_to image_tag(item.product.photo_url, alt:"#{item.product.name}"), product_path(item.product) %></td>
+    <td><%= link_to "#{ item.product.name.capitalize }", product_path(item.product) %></td>
+    <td><%= link_to "#{item.product.merchant.username}", merchant_path(item.product.merchant) %></td>
+    <td><%= form_with model: item do |f| %>
+      <%= f.hidden_field :id, value: @cart.id %>
+      <%= hidden_field_tag :old_quantity, item.quantity %>
+      <% available = item.product.stock_qty + item.quantity %>
+      <%= f.select :quantity, (1..available)%>
+      <%= f.submit "Update quantity" %>
+    <% end %></td>
+    <td>$<%= sprintf('%.2f', item.product.price) %></td>
+    <td>$<%= sprintf('%.2f', (item.quantity * item.product.price)) %></td>
```

Comment:
With more time we can pull some of this logic out of the view and into view helper file